### DISCUSSION TITLE
Consequences of uninstalling the extension

### DISCUSSION THREAD

**Discussion by TheAdminDavid** [rfennell/AzurePipelines#1168] [Q&A] [answered]
Hello,
I have a question regarding the uninstallation of Extensions.
When we uninstall the deprecated Extensions
- Generate Release Notes Build Task (PowerShell)
- StyleCop Runner Build Task
does this break any Build Pipelines?
We currently have them disabled.

Thanks for the help.

**Comment by rfennell** [comment, marked as answer]
There should be no issue in any pipeline along as it does not use the extensions that are being deleted.

I would remove the from the pipelines, not disable them
